frontend-react/src/components/header/SignInOrUser.tsx

```diff
@@ -24,23 +24,24 @@ const SignInOrUser = () => {
         <div className="prime-user-account">
             <span id="emailUser">{user}</span>
             <br />
-            <a
-                href="/"
+            {/* Okta handles our logoutRedirectUri, this should _not_ be an anchor tag! */}
+            <button
```

Comment:
Can we use Trussworks' Button component?  It looks like they have an `unstyled` prop for this: https://trussworks.github.io/react-uswds/?path=/story/components-button--unstyled

Also, let's add a `type="button"` since the default is `type="submit"`, which would cause some weirdness if this were nested in a form for whatever reason :P

Reply:
We could, but this ties back to legacy concerns of how unresponsive Trussworks is in that repo. 😬 Think this ties into the research and discussion around component consolidation - a question of can we trust that library.

Reply:
Right, but we're already using Trussworks' component system in our production code.  I agree that we still need to have that discussion down the line (I'm curious as to what specific issues there are and why we can't implement workaround components like USLink when their components fail us), but I'd argue that a migration down the line would be easier if we have uniformity to start with -- even if that means that we continue using their components in the meantime.

Reply:
Yeah, at a glance, looks like we aren't using nearly as many `<button>`s (notably, on `master` only found 2). I'll change this over; do you mind ticketing the conversion of the other 2 so we can get someone on those and get that uniformity? Saves a bit of trouble on that research ticket :)

Reply:
Sure, I'll add a ticket now and link it to the consolidation ticket!  Thanks!